config.yml

```diff
@@ -6,47 +6,53 @@ CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
 ci_helper_test: &ci_helper_test ci-helper version -e
 MODD_VERSION: &MODD_VERSION 0.5
 
+PYTHON_MANAGED_VERSION: &PYTHON_MANAGED_VERSION
+  - 3.6
+  - 3.7
+  - 3.8
+
+ansible_tests: &ansible_tests
+    cmd:
+      - python --version
+      - pip --version
+      - ansible --version
+      - ansible-playbook --version
+
 ansible:
-  "2.5.5":
+  "2.8":
+    multiple_based_version: true
+    template_vars:
+      VERSION: *PYTHON_MANAGED_VERSION
     build_args:
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
-      VERSION: 2.5.5
-    test_config:
-      cmd:
-        - *ci_helper_test
-        - ansible --version
-        - ansible-playbook --version
-        - python --version
+      ANSIBLE_VERSION: 2.8.8
+    test_config: *ansible_tests
+  "2.9":
+    multiple_based_version: true
+    template_vars:
+      VERSION: *PYTHON_MANAGED_VERSION
+    build_args:
+      ANSIBLE_VERSION: 2.9.4
+    test_config: *ansible_tests
 
 arachni:
   "1.5":
     build_args:
       ARACHNI_VERSION: 1.5.1
       ARACHNI_WEB_UI_VERSION: 0.5.12
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
     test_config:
       cmd:
         - arachni --version
-        - *ci_helper_test
 
 aws:
   "1":
+    multiple_based_version: true
     test_config:
       cmd:
         - aws --version
-        - node --version
-        - npm --version
         - pip --version
-        - pip list | grep boto3
-        - pip list | grep pyyaml
         - pipenv --version
-        - *ci_helper_test
-    build_args:
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
-      NODE_VERSION: 10.16.3-r0
-      PIP_VERSION: 18.1
-      PIPENV_VERSION: 2018.10.13
-      PY_YAML_VERSION: 3.13
+    template_vars:
+      VERSION: *PYTHON_MANAGED_VERSION
```

Comment:
Hi, I'm not sure that we need to have the AWS image in 3 different Pyhton version.

Reply:
That's open to discussion, but I don't want to force every one to use the same python version for their scripts. 
Maybe limiting to 3.7 and 3.8 ?

Reply:
Or just the last stable release ? People are free to continue using old tags.

Reply:
I am not sure on a use case where a user pull the aws image to, in the same time, build this own app (in python) and use the aws-cli. Maybe the principal requirement must be the own of aws-cli : `require Python 2.7, Python 3.4, or a later version to successfully use the AWS CLI.`
So in my mind only the latest version can be necessary ...

Reply:
And as say @clemstoquart , people are free to continue using old tags

Reply:
Oh okay, I'll keep only python 3.8 then

Reply:
> And as say @clemstoquart , people are free to continue using old tags

I've lost the tag mechanism with the old buildbox ^^'